src/index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>lader board</title>
+</head>
+<body>
+  <header>
+    <div class="container">
+      <h1>lader board</h1>
+    </div>
+  </header>
+  <section>
+    <div class="container">
+      <div class="leader-board">
+        <div class="refresh">
+          <span>Refresh the page</span>
+          <button>refersh</button>
+        </div>
+        <div class="board">
```

Comment:
- [ ] Please make sure the refresh text and the button should be outside of the box as on the wireframe.

### Your design
![image](https://user-images.githubusercontent.com/83803414/201978723-45c949c0-03b3-419c-b75b-a9c2a8cbce11.png)

### Wireframe design
![image](https://user-images.githubusercontent.com/83803414/201978695-fa74d258-e16d-4e2c-834c-2402d7903d98.png)